### DISCUSSION TITLE
Is Milvus Standalone 2.0 on the AWS Marketplace already installed on AMI?

### DISCUSSION THREAD

**Discussion by gautinarayanan** [milvus-io/milvus#19422] [Q&A and General discussion]
Also, do we lose performance if we use the standalone version instead of the cluster version on AWS?

![ok](https://user-images.githubusercontent.com/71518939/192130343-759210f0-104b-4321-89d4-e41e2babed3a.jpg)

**Comment by Bennu-Li** [comment]
Yes, Milvus2.0 standalone is already installed on the Milvus AMI. 
And if you have a small data set, you won't lose performance with the standalone version.
